src/main/resources/META-INF/frontend/fcGridHelper/styles.css

```
@@ -0,0 +1,71 @@
+/*-
```

Stylelint (17.14.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)